polyfill/lib/ecmascript.mjs

```diff
@@ -375,7 +373,7 @@ export const ES = ObjectAssign({}, ES2020, {
     // and must be stripped so presence of a calendar doesn't result in interpretation
     // of otherwise ambiguous input as a time.
     const isoStringWithoutCalendar = calendar
-      ? $stringSlice(isoString, 0, isoString.length - calendar.length - 7)
+      ? ES.Call(StringPrototypeSlice, isoString, [0, -(calendar.length + 7)])
```

Comment:
why is this an improvement? this means the intrinsic cache isn't maximally primed by the dep graph.

Reply:
It's for consistency with the rest of the polyfill, which makes use of ES.Call for close alignment with the spec. This PR is not the right place to introduce callBind, although a PR dedicated to that might be appropriate.

Reply:
oh sorry, i missed that your PR was adding it in the first place, nvm.

Reply:
It also occurs to me that #2288 or an immediate followup is quite possibly the place.